### PR TITLE
FIX metadata expression evaluation in doc

### PR DESCRIPTION
**Pull request by fgalan**
Maybe related (not sure) #1634

### CODE REVIEW

File: doc/api.md [hunk 1]
```diff
@@ -607,16 +569,13 @@ expression. In all cases the following data is available to all expressions:
...
 
-Attribute metadata and Static Attribute metadata are available in the **context** under the following convention:
+- measures, as `<AttributeName>`
+- metadata (both for attribute and static attribute) are available in the **context** under the following convention:
 `metadata.<AttributeName>.<MetadataName>` or `metadata.<StaticAttributeName>.<MetadataName>` in a similar way of defined
 for [Context Broker](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#metadata-support)
 
-Moreover, if attribute metadata has an expression metadata attribute value in jexl context it is updated after that
-expression is evaluated. Note that there is no order into metadata structure and there is no warranty about which
-metadata attribute expression will be evaluated first.
```
Comment (fgalan) [inline doc/api.md:618]: > Moreover, if attribute metadata has an expression metadata attribute value in jexl context it is updated after that
expression is evaluated

I think this is redundant with the contents of "Measurement tranformation" section, so it is removed.

> Note that there is no order into metadata structure and there is no warranty about which
metadata attribute expression will be evaluated first.

This is about evaluation ordering, so moved to "Measurement transformation order" section.

File: doc/api.md [hunk 2]
```diff
@@ -607,16 +570,13 @@ expression. In all cases the following data is available to all expressions:
 -   `staticAttributes`: static attributes defined in the device or config group
 
 Additionally, for attribute expressions (`expression`, `entity_name`), `entityNameExp` and metadata expressions
-(`expression`) measures are available in the **context** used to evaluate them.
+(`expression`) the following is available in the **context** used to evalute:
 
-Attribute metadata and Static Attribute metadata are available in the **context** under the following convention:
+- measures, as `<AttributeName>`
+- metadata (both for attribute measurement in the case of NGSIv2 measurments and static attribute) are available in the **context** under the following convention:
```
Comment (fgalan) [inline doc/api.md:576]:
```suggestion
- metadata (both for attribute measurement in the case of NGSI-v2 measurements and static attribute) are available in the **context** under the following convention:
```

**Review by AlvaroVega — APPROVED** [review]
LGTM

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**